src/util/json.cpp

```diff
@@ -160,6 +162,23 @@ std::string messageToJson(const faabric::Message& msg)
     return sb.GetString();
 }
 
+std::string getJsonOutput(const faabric::Message& msg)
+{
+    Document d;
+    d.SetObject();
+    Document::AllocatorType& a = d.GetAllocator();
+    std::string result_ = cppcodec::base64_rfc4648::encode(msg.result());
+    d.AddMember("result", Value(result_.c_str(), result_.size(), a).Move(), a);
+    d.AddMember(
+      "output_data",
+      Value(msg.outputdata().c_str(), msg.outputdata().size(), a).Move(),
+      a);
+    StringBuffer sb;
+    Writer<StringBuffer> writer(sb);
+    d.Accept(writer);
+    return sb.GetString();
+}
```

Comment:
Could you add a small test for this? Can just go alongside the usual `util` tests